classes/issue/Collector.inc.php

```diff
@@ -0,0 +1,460 @@
+<?php
+
+namespace APP\issue;
+
+use Illuminate\Database\Query\Builder;
+use Illuminate\Database\Query\JoinClause;
+use Illuminate\Support\Facades\DB;
+use PKP\core\PKPApplication;
+
+class Collector implements \PKP\core\interfaces\CollectorInterface
+{
+    public const ORDERBY_DATE_PUBLISHED = 'datePublished';
+    public const ORDERBY_LAST_MODIFIED = 'lastModified';
+    public const ORDERBY_SEQUENCE = 'seq';
+    public const ORDERBY_PUBLISHED_ISSUES = 'publishedIssues';
+    public const ORDERBY_UNPUBLISHED_ISSUES = 'unpublishedIssues';
+    public const ORDERBY_GATEWAY_LOCKSS = 'gatewayLockss';
+    public const ORDER_DIR_ASC = 'ASC';
+    public const ORDER_DIR_DESC = 'DESC';
+
+    /** @var DAO */
+    protected $dao;
+
+    /** @var int */
+    protected $count = 30;
+
+    /** @var int */
+    protected $offset = 0;
+
+    /** @var array List of columns to select with query */
+    protected $columns = [];
```

Comment:
@NateWr, the goal of this is to make the column list extensible by outside code, right? I think this is duplicating some query builder functionality we can use. At the top of the `getQueryBuilder` function, we can write...
```
$q = DB::table($this->dao->table, 'i')
    ->select('i.*')
    ...
```
...then any extending code wanting to add a column can write...
```
$q->addSelect('some_other_table.some_other_column AS some_extra_column');
```
(I'm using this successfully in the reviewer extensions to the user stuff.)

Reply:
Yes, I think that we can remove the use of `$columns`. I think it was probably related to some conditional joins or something, but shouldn't be needed.